### PR TITLE
LITE-21474 Added optional cqrs-id argument to consume command

### PR DESCRIPTION
**Pull request by maxipavlovic**

### CODE REVIEW

**Comment by maxipavlovic** [comment]
It's unreasonably hard to follow coverage rule in this case at this moment, so sonar report fails, though all integration tests are fine!

**Review by net-free — CHANGES_REQUESTED** [review]

**Review by d3rky — APPROVED** [review]

File: dj_cqrs/transport/mock.py [hunk 1]
```diff
@@ -9,5 +9,5 @@ def produce(payload):
         return TransportMock.consume(payload)
 
     @staticmethod
-    def consume(payload):
+    def consume(payload=None, **kwargs):
```
[the remark this answers is not in the document]
Comment (maxipavlovic) [reply dj_cqrs/transport/mock.py:12]: I think it's better to change as they may break now without the changes.

File: dj_cqrs/management/commands/cqrs_consume.py [hunk 2]
```diff
@@ -12,17 +12,30 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('--workers', '-w', help='Number of workers', type=int, default=0)
+        parser.add_argument(
```
[the remark this answers is not in the document]
Comment (maxipavlovic) [reply dj_cqrs/management/commands/cqrs_consume.py:15]: At first, I decided to skip validation. But as you wrote and the more I think on that, the more it makes sense. I will add validation.

[the remark this answers is not in the document]
Comment (maxipavlovic) [reply dj_cqrs/management/commands/cqrs_consume.py:16]: Fixed

**Review by net-free — APPROVED** [review]

**Review by zzzevaka — APPROVED** [review]
